packages/@vue/cli/lib/util/nodeVersion.js

```diff
@@ -0,0 +1,5 @@
+exports.minNode = function (major, minor = 0, patch = 0) {
+  const parts = process.versions.node.split('.').map(v => parseInt(v))
```

Comment:
could use destructuring and give part names like const [major,minor,patch] = ...

i’m aware these are already taken, just suggesting.